src/posix.mak

```diff
@@ -420,6 +422,9 @@ all: dmd
 dmd: $G/dmd $G/dmd.conf
 .PHONY: dmd
 
+$(GENERATED)/build: build.d $(HOST_DMD_PATH)
```

Comment:
```suggestion
$(RUN_BUILD): build.d $(HOST_DMD_PATH)
```

Reply:
`$(RUN_BUILD)` is both the build exectuable and the environment variable arguments, which doesn't make sense here because the environment variables are not files to be put in a makefile target.